recipes/mender-monitor/debian-1.1.x/control

```diff
@@ -0,0 +1,19 @@
+Source: mender-monitor
+Maintainer: Mender Team <mender@northern.tech>
+Section: admin
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender-monitor
+Architecture: all
+Depends: bash, lmdb-utils, mender-auth | mender-client (>=2.5.0), ${misc:Depends}
+Description: Mender Monitor client
+ Mender Monitor is a Mender add-on which enables the reporting of certain
+ device events to the Mender server, such as log events, service files statuses
+ and D-Bus signals.
+ 
+Package: mender-monitor-demo
+Architecture: all
+Depends: mender-monitor (= ${binary:Version}), ${misc:Depends}
+Description: Adds demo functionality to Mender Monitor.
```

Comment:
I am sure we can do better than "Adds demo functionality to Mender Monitor" since it really adds demo checks and subsystems (at least last time I were there)